docs/deployment/nativeaot.md

```diff
@@ -139,6 +139,85 @@ Use the following checklist to help you adapt your app to Native AOT deployment
 > - Review and correct any trimming or AOT warnings. For more information, see [Introduction to trim warnings](/dotnet/core/deploying/trimming/fixing-warnings) and [Introduction to AOT warnings](/dotnet/core/deploying/native-aot/fixing-warnings).
 > - Thoroughly test your app.
 
+## Native AOT diagnostic support on iOS-like platforms
```

Comment:
What do you mean by iOS-like platforms?

Reply:
It is an *unofficial* term used to cover the following platforms:
- iOS
- iOS Simulator
- Mac Catalyst
- tvOS
- tvOS Simulator

However, since MAUI does not support tvOS, it might be better to explicitly refer to only iOS and Mac Catalyst